src/middlewares/accommodation.middleware.js

```diff
@@ -55,6 +57,23 @@ const Accommodation = {
     }
     return next();
   },
+
+  async isAccommodationAvailable(req, res, next) {
+    const { AccommodationId } = req.body;
+    if (await BookingService
+      .checkAccommodationAvailability(AccommodationId)) {
+      return next();
+    }
+    return sendResult(res, 400, 'The Accommodation is booked out');
+  },
+
+  async accommodationExists(req, res, next) {
```

Comment:
- This look like the responsibility of a service - `AccommodationService`.
- Create an `accommodation.service.js` file and export these methods in an object. When we get to refactoring, we'll merge them together.
- Also change the method name to `getAccommodationById`. Since that's basically what the method is doing. `get`ting an accomodation, by the `ID`.